tests/unit/test_charm.py

```diff
@@ -4,64 +4,14 @@
 # Learn more about testing at: https://juju.is/docs/sdk/testing
 
 import unittest
-from unittest.mock import Mock
 
-from ops.model import ActiveStatus
 from ops.testing import Harness
 
-from charm import OperatorTemplateCharm
+from charm import TraefikIngressCharm
 
 
-class TestCharm(unittest.TestCase):
+class TestTraefikIngressCharm(unittest.TestCase):
     def setUp(self):
-        self.harness = Harness(OperatorTemplateCharm)
+        self.harness = Harness(TraefikIngressCharm)
```

Comment:
I can only assume there is more to come here :sweat_smile:

Reply:
I never said I would provide tests in alpha ;-)
The TODO promises a spec of which integration tests are needed.

Reply:
Specifically, I am not writing brittle unit tests when we know we will change the relation interface and likely library significant with @johnsca work ongoing to stabilize the `istio-pilot` lib.